### PR TITLE
Add compat data to api.WheelEvent

### PR DESCRIPTION
**Pull request by luciancic**
[WheelEvent](https://developer.mozilla.org/docs/Web/API/WheelEvent)

### CODE REVIEW

File: api/WheelEvent.json [hunk 1]
```diff
@@ -50,45 +50,47 @@
...
+              "version_added": true
             },
             "safari": {
-              "version_added": null
+              "version_added": true,
+              "notes": "Safari does not support <code>WheelEvent</code> but returns an object from <code>window.WheelEvent</code> nonetheless."
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": true,
```
Comment (maboa) [inline api/WheelEvent.json]: Although it's probably safe to assume `true`, I'm seeing `safari_ios` as `null`.

File: api/WheelEvent.json [hunk 2]
```diff
@@ -305,45 +257,45 @@
...
-              "version_added": null
+              "version_added": "18"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": true
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```
Comment (maboa) [inline api/WheelEvent.json]: I'm seeing this as `9` which is clearly odd since `deltaX` has it as `false`.

File: api/WheelEvent.json [hunk 3]
```diff
@@ -254,45 +206,45 @@
...
-              "version_added": null
+              "version_added": "17"
             },
             "firefox_android": {
-              "version_added": null
+              "version_added": "17"
             },
             "ie": {
-              "version_added": null
+              "version_added": "9"
```
Comment (maboa) [inline api/WheelEvent.json]: Assume we're not adding a `notes` field here for `ie`?

File: api/WheelEvent.json [hunk 4]
```diff
@@ -305,45 +257,45 @@
...
```
Comment (maboa) [inline api/WheelEvent.json]: Assume we're not adding a notes field here for ie?

File: api/WheelEvent.json [hunk 5]
```diff
@@ -356,45 +308,45 @@
...
```
Comment (maboa) [inline api/WheelEvent.json]: Assume we're not adding a notes field here for ie?

File: api/WheelEvent.json (hunk 1 above)
Comment (lucian95) [reply api/WheelEvent.json]: Seeing as it returns an object from `window.WheelEvent` but does not actually work. I think marking it `false` and adding a note is a better way to go.

File: api/WheelEvent.json (hunk 2 above)
Comment (lucian95) [reply api/WheelEvent.json]: Yes, it's odd. I thought that it made no sense to have support on a property when the basic support is false.

File: api/WheelEvent.json (hunk 3 above)
Comment (lucian95) [reply api/WheelEvent.json]: Thanks for catching it! I added it

File: api/WheelEvent.json (hunk 4 above)
Comment (lucian95) [reply api/WheelEvent.json]: ditto

File: api/WheelEvent.json (hunk 5 above)
Comment (lucian95) [reply api/WheelEvent.json]: ditto

**Comment by wbamberg** [comment]
screen shot 2018-08-13 at 8 29 27 pm

**Pull request merged**